aip/general/0162.md

```diff
@@ -228,178 +189,83 @@ there are two potential variants:
 - Child resources where each child resource is a child of _a single revision
   of_ the parent resource.
 
-If a child resource is a child of a single revision, the child resource's name
-**must** always explicitly include the parent's revision ID:
+APIs **should not** include multiple levels of resources with revisions, as
+this quickly becomes difficult to reason about.
 
-    publishers/123/books/les-miserables@c7cfa2a8/pages/42
+### Standard methods
 
-In `List` requests for such resources, the service **should** default to the
-latest revision of the parent if the user does not specify one, but **must**
-explicitly include the parent's revision ID in the `name` field of resources in
-the response.
+Any standard methods **must** implement the corresponding AIPs ([AIP-131][],
+[AIP-132][], [AIP-133][], [AIP-134][], [AIP-135][]).
```

Comment:
The two links for [API-133] and [API-134] aren't working --- they need their footnote-style links at the bottom of the file.

Reply:
Actually we don't need hard-coded links to AIPs because the site-generator adds them for you.

So I'll just make this naked text to make it less error-prone.